network/p2p/inspector/validation/control_message_validation_inspector.go

```diff
@@ -456,6 +458,7 @@ func (c *ControlMsgValidationInspector) truncateIHaveMessageIds(rpc *pubsub.RPC)
 		c.performSample(p2pmsg.CtrlMsgIHave, uint(totalMessageIDs), uint(sampleSize), func(i, j uint) {
 			messageIDs[i], messageIDs[j] = messageIDs[j], messageIDs[i]
 		})
+		ihave.MessageIDs = messageIDs[:sampleSize]
```

Comment:
```suggestion
		ihave.MessageIDs = messageIDs[:maxMessageSize]
```

Reply:
I would rename this variable to be more clear:

before:
`sampleSize := c.config.IHaveRPCInspectionConfig.MaxMessageIDSampleSize`

after:
`maxMessageSize := c.config.IHaveRPCInspectionConfig.MaxMessageIDSampleSize`

Reply:
`maxMessageSize` implies the size of a single message, while `sampleSize` implies the size of the sample we are taking as we are sampling the rpcs or message ids. So the line` ihave.MessageIDs = messageIDs[:sampleSize]` can clearly be inferred as "take a sample of sampleSize" which more clearly describes the action being taken.